Fonts.css

```diff
@@ -0,0 +1,452 @@
+  justify-content: center;
+  min-height: 18.75rem;
+  grid-gap: 7.5rem;
+}
+.grid3 {
+  display: grid;
+  padding-top: 1.25rem;
+  padding-bottom: 3.75rem;
+  grid-template-columns: 50rem;
+  justify-content: right;
+  min-height: 31.25rem;
+  grid-gap: 50rem;
+  width: auto;
+}
+.gridwrapper_5 {
+  display: grid;
+  grid-template-columns: 22.5rem 22.5rem 22.5rem 22.5rem;
+  justify-content: center;
+  grid-auto-flow: row dense;
+  grid-gap: 1.5rem;
+}
+.box1-contact{
+
+  height: auto;
+  width: auto;
+  justify-content: center;
+}
+.gridwrapper_header {
+  display: grid;
+  grid-template-columns: 3fr 3fr 3fr;
+  justify-content: center;
+  grid-auto-flow: row dense;
+  grid-gap: 1.5rem;
+}
+nav {
+  width: auto;
+font-family: "oskar";
+font-weight: bold;
+margin: 2rem;
+}
+nav ul {
+  margin: 2rem;
+}
+nav a{
+  margin: 0.25rem;
+  padding: 0.25rem;
+}
+ input, button,  select {
+display: block;
+font-family: inherit;
+font-size: 100%;
+width: 18.75rem;
+height: 1.563rem;
+box-sizing: border-box;
+}
+textarea{
+  width: 25rem;
+  height: 6.25rem;
+
+}
+button{
+
+  width: auto;
+  height: auto;
+
+}
+input [type=submit]{
+
+  width: 100%;
+  background-color: #777;
+}
+fieldset input{
+
+  display: inline;
+}
+
+.checkbox input:checked ~ .checkmark {
+  background-color: #777;
+}
+
+.gridwrapper_contactform{
+  display: grid;
+  grid-template-columns: 3fr 3fr 3fr;
+  justify-content: center;
+  grid-auto-flow: row dense;
+  grid-gap: 1.5rem;
+}
+.gridwrapper_5 input{
+
+}
+.gridwrapper_5 button{
+
+}
+.gridwrapper_5 textarea{
+
+}
+.wrapper-contact{
+  display: flex;
+  grid-column: 2;
+  justify-content:center;
+}
+.hidden{
+  position: relative;
+  left: -10000px;
+  top:auto;
+}
+.hidden:focus{
+  left: 0;
+}
+
+.footer1{
+display: block;
+width: auto;
+height: auto;
+}
+.footer2{
+  display: block;
+  width: auto;
+  height: auto;
+}
+.footer3{
+  display: block;
+  width: auto;
+  height: auto;
+}
+
+
+
+
+.contweh {
+  font-family: Arial, Helvetica, sans-serif;
+  max-width: 1050px;
+  margin: auto;
+}
+
+.flex_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+}
+
+.left_box {
+  flex: 3;
+}
+
+.right_box {
+  flex: 1;
+  margin: 1rem;
+}
+
+
+.left_box img {
+  width: 100%;
+  max-height: 280px;
+}
+
+
+.dogb img {
+  width: 100%;
+  height: 150px;
+}
+
+
+
+.dogb button {
+  display: block;
+  width: 100%;
+  font-size: large;
+  background-color: rgb(50, 128, 50);
+  color: white;
+}
+
+.grid_dog {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 0.4rem;
+}
+
+.grid_dog1 {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 0.4rem;
+}
+
+
+.right_column {
+  display: flex;
+  flex-direction: column;
+
+  justify-content: center;
+}
+.border{
+  border: 3px dotted orange;
+}
+
+
+.sright img {
+  width: 100%;
+  max-height: 170px;
+}
+
+
+@media(min-width: 501px) and (max-width: 768px) {
```

Comment:
it's generally not a good idea to use both min width and max width, things might not behave as youwant